docs/content/latest/admin/yb-admin.md

```diff
@@ -1515,6 +1515,38 @@ yb-admin \
     list_cdc_streams
 ```
 
+#### delete_cdc_stream <stream_id> [force_delete]
+
+Deletes underlying CDC stream for the specified YB-Master servers.
```

Comment:
I think we should specify that this should only be used for deleting old bootstrapped streams that were never fully initialized, and not for use on regular xcluster replication setups (although there are now safeguards in place that should prevent that)